### ISSUE TITLE
problem with CI when an other PR updates the composer dependencies

### ISSUE THREAD

**Issue by individual-it** [owncloud/ocis-php-sdk#149]
The PR #148 changed the `phpunit` dependency, but now other builds fail:
https://drone.owncloud.com/owncloud/ocis-php-sdk/832/10/3

```
+ composer install
Installing dependencies from lock file (including require-dev)
Verifying lock file contents can be installed on current platform.
Warning: The lock file is not up to date with the latest changes in composer.json. You may be getting outdated dependencies. It is recommended that you run `composer update` or `composer update <package name>`.
- Required (in require-dev) package "phpunit/phpunit" is in the lock file as "10.5.3" but that does not satisfy your constraint "^9.6".
This usually happens when composer files are incorrectly merged or the composer.json file is manually edited.
Read more about correctly resolving merge conflicts https://getcomposer.org/doc/articles/resolving-merge-conflicts.md
and prefer using the "require" command over editing the composer.json file directly https://getcomposer.org/doc/03-cli.md#require-r
```

**Comment by phil-davis** [comment]
But that PR has not been merged????

**Comment by individual-it** [comment]
Yes, that made me think that the problem is the order of execution. The docs pipeline runs too early, before the cache is rebuild and gets whatever was there in the cache before.

I hope it will be fixed by #150
